x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.tsx

```diff
@@ -87,10 +87,12 @@ export const EditOutputFlyout: React.FunctionComponent<EditOutputFlyoutProps> =
   const { kafkaOutput: isKafkaOutputEnabled, remoteESOutput: isRemoteESOutputEnabled } =
     ExperimentalFeaturesService.get();
   const isRemoteESOutput = inputs.typeInput.value === outputType.RemoteElasticsearch;
+  // Remote ES output not yet supported in serverless
+  const isTraditionalBuildFlavor = useKibanaBuildFlavor() === 'traditional';
```

Comment:
We also use `cloudSetup.isServerlessEnabled` to check if it's serverless, I am wondering if there is one that is better than the other, and we should probably stick to one

Reply:
oh right, I forgot about that, it is simpler to use that

Reply:
done